remote-api/src/main/java/org/alfresco/rest/api/impl/ContentStorageInformationImpl.java

```diff
@@ -74,6 +80,33 @@ public ContentStorageInfo getStorageInfo(String nodeId, String contentPropName,
         return storageInfo;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean requestArchiveContent(String nodeId, String contentPropName,
+                                         ArchiveContentRequest archiveContentRequest)
+    {
+        final NodeRef nodeRef = new NodeRef(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, nodeId);
+        final QName propQName = getQName(contentPropName);
+        return contentService.requestSendContentToArchive(nodeRef, propQName, archiveContentRequest.getArchiveParams());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean requestRestoreContentFromArchive(String nodeId, String contentPropName,
+                                                    RestoreArchivedContentRequest restoreArchivedContentRequest)
+    {
+        final NodeRef nodeRef = new NodeRef(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, nodeId);
+        final QName propQName = getQName(contentPropName);
+        final Map<String, Serializable> restoreParams = restoreArchivedContentRequest.getRestorePriority() == null ?
```

Comment:
Confused here. EXPIRY_DAYS seems to be missing. Both from this map and the restoreArchivedContentRequest. Wouldn't it be better to add a Map like the archiveContentRequest?

Reply:
Following conversation, I understand now that EXPIRY_DAYS are not being used as a requestParam as this does not allow the request to be cloud agnostic. Instead we are using config on the connector side.